tests/unit-tests/serve_test.go

```diff
@@ -0,0 +1,62 @@
+package tests
+
+import (
+	"net/http/httptest"
+
+	diddocServices "github.com/cheqd/did-resolver/services/diddoc"
+	"github.com/cheqd/did-resolver/types"
+	"github.com/labstack/echo/v4"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Content/Accept encoding checks", func() {
```

Comment:
Do we need to add unit tests for testing any accept-encoding.

Reply:
Hey! Yes, we've already added unit tests. Thanks for the suggestion